### DISCUSSION TITLE
Add Print Function

### DISCUSSION THREAD

**Discussion by Cruhrup** [astashov/liftosaur#220] [Ideas]
This way we can output customized plain text for things as well as help with debugging scripts. Attached screenshot of what it may look like for a program after setting the State Variable 

![printfunction_example](https://github.com/user-attachments/assets/884ce514-81c5-4f34-81c5-505aaf6ef56a)

**Comment by steshaw** [comment]
It might be worth adding logging with debug levels so that people aren't overwhelmed with output when running a program with a lot of debug prints.

**Comment by yrinkel** [comment]
This can probably be closed now that it’s been implemented? 😊

**Reply by astashov** [reply]
Good point

**Comment by astashov** [comment]
Added `print` function
